Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,7 +38,51 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
+            if (!string.IsNullOrEmpty(message))
```

Comment:
Prefer to state, rather than negate.
Prefer to exit/fail early. 
So instead:
```cs
if (string.IsNullOrEmpty(message)) return String.Empty;

                var sb = new StringBuilder();
                var messages = message.Split(Environment.NewLine);
                var longestString = "";
                for (int i = 0; i < messages.Length; i++)
                {
                    if (messages[i].Length > longestString.Length)
                        longestString = messages[i];
                }
                var whiteSpace = "";
                var length = longestString.Length + padding * 2;
                sb.Append($"+{whiteSpace.PadRight(length, '-')}+{Environment.NewLine}");
                BuildWhitespace(sb, whiteSpace, padding, length);
                BuildMessage(sb, messages, padding, longestString.Length);
                BuildWhitespace(sb, whiteSpace, padding, length);
                sb.Append($"+{whiteSpace.PadRight(length, '-')}+{Environment.NewLine}");

                return sb.ToString();
```

Reply:
It's better, because you no longer have code nested in an if statement, you take boring code and run it first, the interesting part is not mixed with the boring part.

Reply:
inverted the if statement to reflect this